packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -455,9 +456,25 @@ export interface IPerfSignalReport {
 	signalTimestamp: number;
 
 	/**
-	 * Expected Signal Sequence to be received.
+	 * Signal we will trace for roundtrip latency.
+	 */
+	roundTripSignalSequenceNumber: number | undefined;
+
+	/**
+	 * Last signal traced for roundtrip latency.
+	 */
+	previousRoundTripSignalSequenceNumber: number | undefined;
```

Comment:
I've looked at the uses of `previousRoundTripSignalSequenceNumber` and it does not appear to be the previous round trip number. It is the lower bound of the upcoming-now-current tracking group.
When latency event is sent, `previousRoundTripSignalSequenceNumber` is set to the last tracked + 1.
When things are initialized, `previousRoundTripSignalSequenceNumber` is set to the current number.

Reply:
Right, make sense. Updated to just be the lower bound of the upcoming/current tracking group.